simple_certmanager/utils.py

```diff
@@ -75,6 +75,12 @@ def pretty_print_certificate_components(x509name: x509.Name) -> str:
     return ", ".join(bits)
 
 
+def pretty_print_serial_number(x509sn: int):
+    num_bytes = (x509sn.bit_length() + 7) // 8
+    byte_values = x509sn.to_bytes(num_bytes, byteorder="big")
```

Comment:
```suggestion
    byte_values = x509sn.to_bytes(num_bytes or 1, byteorder="big")
```
makes it work for all naturals.

Reply:
Yikes, shouldn't have missed this. Hypothesis would've warned me..